fast-list.js

```diff
@@ -57,8 +57,8 @@ function FastList(source) {
     switchWindow: 0
   };
 
-  this.geometry.headerHeight = source.sectionHeaderHeight();
-  this.geometry.itemHeight = source.itemHeight();
+  this.geometry.headerHeight = source.getSectionHeaderHeight();
+  this.geometry.itemHeight = source.getItemHeight();
   this.updateContainerGeometry();
```

Comment:
Let's move this call to the top of the `schedule.mutation` block at the end of the function.
It's clearer, makes sense since it might trigger a reflow.
But you'll probably need to move 1 test :)